solidity/contracts/bridge/Bridge.sol

```diff
@@ -1065,7 +1071,13 @@ contract Bridge is Ownable {
             "There is a pending redemption request from this wallet to the same address"
         );
 
-        uint64 treasuryFee = redemptionTreasuryFee;
+        // No need to check whether `amount - treasuryFee - txMaxFee > 0`
+        // since the `redemptionDustThreshold` should force that condition
+        // to be always true.
+        // TODO: Is it okay to lose the precision here?
```

Comment:
Solidity will truncate (round down) the fee but we subtract it from the requested `amount` to calculate `pendingRedemptionsValue` so I think we are not losing any dust. Some rounding would need to happen anyway. Am I missing something?

Reply:
Just an old TODO I left at the beginning and forgot to remove it at the end. Removed it https://github.com/keep-network/tbtc-v2/pull/136/commits/5657086ce3a129119844fcf54eb46017bd3fa0e5